### ISSUE TITLE
[BUG]: Unresolved imports in docs

### ISSUE THREAD

**Issue by AbhijithGanesh** [piccolo-orm/piccolo#293]
In the [given](https://piccolo-admin.readthedocs.io/en/latest/asgi/index.html) page you can see imports for starlette mount is not imported here.

**Comment by AbhijithGanesh** [comment]
Yes, I would like to make this minor documentation fix.

**Comment by dantownsend** [comment]
@AbhijithGanesh Yes, what do I need to do? Just tag this issue with `hacktoberfest`?

**Comment by AbhijithGanesh** [comment]
I think you need to register the repository.

**Comment by AbhijithGanesh** [comment]
@dantownsend can you please review my changes?
